subsys/bluetooth/host/Kconfig

```diff
@@ -601,6 +601,13 @@ config BT_ID_MAX
 	  Maximum number of supported local identity addresses. For most
 	  products this is safe to leave as the default value (1).
 
+config BT_DIRECTION_FINDING
```

Comment:
I think this option should be BT_DF (since we have BT_CTLR_DF).

And probably when create an API we will also use `_df_` there as well, they should align.

Reply:
config changed to BT_DF.

> And probably when create an API we will also use _df_ there as well, they should align.
I will remember to add _df_ into public API names also.